### PR TITLE
add S3 objects for generated data

### PR DESCRIPTION
**Pull request by laughedelic**
Right now in `src/main` we don't have the address of the generated data.

### CODE REVIEW

**Comment by laughedelic** [comment]
The problem:
- you want these addresses in `Compile` scope
- but the filters that provide this information are defined in the `Test` scope

A simple solution that doesn't involve changes in the db.rnacentral code:
- set these addresses first
- add another bundle (in `Test`) that just copies things from the last filter to the given locations defined above

**Comment by eparejatobes** [comment]
Fine. We should polish db.RNACentral a bit at some time, though.

File: src/main/scala/release.scala [hunk 1]
```diff
@@ -4,13 +4,7 @@ import ohnosequences.awstools.s3._
...
-  lazy val prefix: S3Folder =
-    S3Folder("resources.ohnosequences.com", metadata.organization)/metadata.artifact/metadata.version/
-
-  // TODO these should have a value!
-  lazy val fastaS3:    S3Object = prefix/ ???     // dropInconsistentAssignments.output.fasta.s3
-  lazy val id2taxasS3: S3Object = prefix/ ???     // dropInconsistentAssignments.output.table.s3
-  lazy val blastDBS3:  S3Folder = prefix/ ??? /   // dropInconsistentAssignmentsAndGenerate.s3
+  lazy val fastaS3:    S3Object = s3prefix/ "release" / s"${dbName}.fasta"
+  lazy val id2taxasS3: S3Object = s3prefix/ "release" / s"${dbName}.csv"
+  lazy val blastDBS3:  S3Folder = s3prefix/ "release" / "blastdb" /
```
Comment (laughedelic) [inline src/main/scala/release.scala:9]: @eparejatobes check the names format, please

**Comment by laughedelic** [comment]
This is done. Waiting for @eparejatobes' approvement. In the end I made a simple method that you call locally and it just copies those files in S3.

File: src/main/scala/release.scala (hunk 1 above)
Comment (eparejatobes) [inline src/main/scala/release.scala:9]: why `"release"` there?

Comment (laughedelic) [inline src/main/scala/release.scala:9]: Because `s3prefix/` contains other subfolders with results of the filters

Comment (eparejatobes) [inline src/main/scala/release.scala:9]: OK fair enough

**Comment by eparejatobes** [comment]
LGTM

**Pull request merged**